schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+    id INT PRIMARY KEY NOT NULL, 
```

Comment:
[OPTIONAL]
- Great use of the `primary key` and `not null` constraints 👍 
I can see that you're currently hard coding the `id` of each record/row being added. I would suggest that you add the `GENERATED ALWAYS AS IDENTITY` constraint to the `id` column. This will help you to generate sequential integers that cannot be altered when the record is being created.
You can go through this [guide](https://www.sqltutorial.org/sql-identity/#:~:text=The%20GENERATED%20ALWAYS%20generates%20sequential,integers%20for%20the%20identity%20column.) for more details. 😉

Reply:
- Thank you very much, @DammyShittu. I have made changes to the project with reference to the helpful content you provided. I really appreciate the insightful review. This is a lesson I will surely apply in future projects.
- HAPPY CODING!!! :computer: :smiley: